paimon-core/src/main/java/org/apache/paimon/stats/FieldStatsArraySerializer.java

```diff
@@ -65,6 +77,7 @@ public FieldStatsArraySerializer(
                         .toArray(InternalRow.FieldGetter[]::new);
         this.indexMapping = indexMapping;
         this.converterMapping = converterMapping;
+        this.skipFieldIndex = skipFieldIndex;
```

Comment:
It is OK to just name this `fieldsWithDefaultValue`?

Reply:
Can we use skipStatField instead? This might be more generic in FieldStatsArraySerializer ,FieldStatsArraySerializer  may don't need care the default vale concept  of table.

Reply:
ignore it , `FieldStatArraySerializer` should be related to the fields of a table, so it should still be within the scope of the concept of a table.